static/app/views/eventsV2/savedQuery/index.tsx

```diff
@@ -321,6 +332,19 @@ class SavedQueryButtonGroup extends React.PureComponent<Props, State> {
     );
   }
 
+  renderDiscoverQueryMenu() {
+    const menuOptions: React.ReactNode[] = [];
+    menuOptions.push(
+      <MenuItem
```

Comment:
Something weird happens when you try to add a query to the `default-overview` dashboard. I end up having to click save twice and that adds duplicate widgets to my dashboard